### DISCUSSION TITLE
Quest Testing - Update

### DISCUSSION THREAD

**Discussion by KVonGit** [textadventures/quest#1260] [Developing Quest]
We just updated the testing build to fix #1259 

[![Quest Testing Central](https://github.com/user-attachments/assets/f684aff5-8b86-454c-830e-c747993dcad5)](https://textmisadventures.github.io/quest/ "https://textmisadventures.github.io/quest/")
[Quest Testing Central | Test the desktop version](https://textmisadventures.github.io/quest/#test-the-desktop-version "https://textmisadventures.github.io/quest/#test-the-desktop-version")

---
[All open pull requests](https://github.com/textadventures/quest/pulls) are included in the testing build.

**Comment by ghost** [comment]
The testing website is now being hosted from GitHub Pages[^1].

https://textmisadventures.github.io/quest/

[^1]: The original site ~~is also still up~~ now forwards here.

**Comment by KVonGit** [comment]
[![Quest Testing Central](https://github.com/user-attachments/assets/f684aff5-8b86-454c-830e-c747993dcad5)](https://textmisadventures.github.io/quest/ "https://textmisadventures.github.io/quest/")
[Quest Testing Central | Test the desktop version](https://textmisadventures.github.io/quest/#test-the-desktop-version "https://textmisadventures.github.io/quest/#test-the-desktop-version")

---
We just updated the testing build.

- Handle alias including `.` when `game.multiplecommands` is `true`
- Fix error in `ProcessTextCommand_Element`

**Comment by ghost** [comment]
[![391309181-f684aff5-8b86-454c-830e-c747993dcad5](https://github.com/user-attachments/assets/091df3c4-f898-4e2e-9504-5974729a7c05)](https://textmisadventures.github.io/quest/ "https://textmisadventures.github.io/quest/")

[Quest Testing Central](https://textmisadventures.github.io/quest/ "https://textmisadventures.github.io/quest/")

---
We just updated [the testing build](https://textmisadventures.github.io/quest/#test-the-desktop-version "https://textmisadventures.github.io/quest/#test-the-desktop-version") again. [^1]

[^1]: The WebPlayer app on [the testing site](https://textmisadventures.github.io/quest/ "https://textmisadventures.github.io/quest/") has also been updated. [^2]

[^2]: This is usually the case, but I usually fail to mention it.

Nothing new was added. It was all bug fixes:
- fixed issue with POV object in games created in Spanish
- fixed bug with AGAIN command when the template was not all lower-case
- copied the WebEditor's favicon to the WebPlayer

---
Happy gaming!

**Comment by ghost** [comment]
Seasons greetings!

---
The testing build has been updated.

Direct download: [Quest5.9.9123.20935-Debug.zip](https://github.com/TextMisadventures/quest/releases/download/v5.9.9123.20935-alpha/Quest5.9.9123.20935-Debug.zip)

https://github.com/TextMisadventures/quest/releases/tag/v5.9.9123.20935-alpha

---
**Changes since the last build:**
- Added `MovePlayer` function in [1d0764d](https://github.com/TextMisadventures/quest/commit/1d0764dc2bd4f1d312b44f4acc9f669391dd84e7)
  -  This includes editor controls
- `ChangePOV` will now throw an error if the object has no parent in [29eb6b7](https://github.com/TextMisadventures/quest/commit/29eb6b742d74188986ffe87257f282efff6e3646)
- Changed link created by `ShowMenu` back to using `ASLEvent` in [29eb6b7](https://github.com/TextMisadventures/quest/commit/132f5ce788938f9ce1d2183d771a34ecc227db80)
- Updated `ShowMenu` to print the choice made by the player in [132f5ce](https://github.com/TextMisadventures/quest/commit/132f5ce788938f9ce1d2183d771a34ecc227db80)
  - submitted by @Pertex 
  - The text is preceded by `>> `
    - This uses the new `ShowMenuResponsePrompt` Template
- `&nbsp;` is now replaced with a space in the transcript text (in [c6eabc4](https://github.com/TextMisadventures/quest/commit/c6eabc45d06d09f0b2519a70e994f0017ab08696))

---
Happy gaming, merry Christmas, and happy holidays!

[Quest Testing Central](https://textmisadventures.github.io/quest/ "https://textmisadventures.github.io/quest/")
[![391309181-f684aff5-8b86-454c-830e-c747993dcad5](https://github.com/user-attachments/assets/091df3c4-f898-4e2e-9504-5974729a7c05)](https://textmisadventures.github.io/quest/ "https://textmisadventures.github.io/quest/")
